### DISCUSSION TITLE
B&G Depth Transducer

### DISCUSSION THREAD

**Discussion by Robifz** [bareboat-necessities/lysmarine_gen#286] [General]
I would like to connect my old B&G Depth Transducer. Is there anything I can use that can decode the electrical signal from the transducer. I would assume it is a propriety B&G signal.
![A+T Depth Sensor](https://user-images.githubusercontent.com/126383859/221401817-cc003deb-88f5-4c63-b696-ec98525847e1.jpg)

**Comment by mgrouch** [comment]
Might be this can be helpful:

https://marinehowto.com/new-depth-sounder-with-an-old-transducer/

**Comment by mgrouch** [comment]
Actisense® DST-2
can decode analog depth signals into NMEA. 

You need to know frequency of your transducer to match it with decoder.

**Comment by mgrouch** [comment]
Found some code which does it (for NASA depth, but principle should be same). 

Here is the project

https://github.com/pljakobs/NASAMarineDuetDriver

**Comment by mgrouch** [comment]
And another project doing the same:

https://open-boat-projects.org/en/nmea2000-fuer-nasa-clipper-duet-echolot-log/

With the code at: 
https://github.com/speters/ClipperDuet2N2k
